src/utils/color-utils.js

```diff
@@ -60,6 +60,25 @@ const colorUtils = {
     const current = getCurrentTheme(Theme);
     return (current && current.properties.fontFamily) || 'inherit';
   },
+  getFadedColor(color) {
+    const percent = -0.15;
+    let f; let R; let B; let G;
+    const t = 0; // percent < 0 ? 0 : 255
+    const p = percent * -1; // percent < 0 ? percent * -1 : percent
```

Comment:
You want to keep these comments in case we want to move back to percentage? Also why not set percent to a positive value and use that one?

Reply:
That was my thought, but it might be unnecessary.